docs/endpoints/posts.md

```diff
@@ -168,31 +168,41 @@ This will automatically create or update the corresponding signup.
 POST /api/v3/posts
 ```
 
-- **campaign_id**: (int) required.
-  The Campaign ID of the campaign that the user's post is associated with.
-- **type**: (string) required.
-  The type of post submitted. Must be one of the following types: `photo`, `voter-reg`, `text`, `share-social`. `share-social` posts will be auto-accepted unless an admin sets a custom `status`.
-- **action**: (string) required.
-  Describes the bucket the action is tied to. A campaign could ask for multiple types of actions throughout the life of the campaign.
-- **quantity**: (int|nullable) optional.
+Required params:
+
+- **action_id**: (int) required without action or campaign_id.
+  The action ID of the action that the user's post is associated with.
+- **file**: (multipart/form-data) required for `photo` posts.
+  File to save of post image.
```

Comment:
I actually don't think this is true. If we do want to make this validation, we might have to add in `type` as a requirement again in order to do this?

Reply:
Resolved [here](https://github.com/DoSomething/rogue/pull/837/files#r261276786).